ADA/ada.qmd

```diff
@@ -19,57 +17,194 @@ It brings together:
 *	a form for requesting access to data
 *	a support section - with walkthrough instructions
 *	a collection of data reports and dashboards
-*	a news and updates area.
+*	a news and updates area
+
+As long as you're connected to the network - it's available to anyone who works for the Department of Education. 
+
+The ADA project is currently in its onboarding phase, which is expected to continue until 2025. At the moment, teams can onboard voluntarily until **???**. Following this date, the remaining teams will have their data migrated for them. 
+Data migration will replace three legacy systems: 
+
+* Pupil Data Repository (PDR)
+* Analysis and Modelling (A&M)
+* Enterprise Data and Analysis Platform (EDAP1)
+
+See these [Databricks support notebooks](https://adb-2220072380334347.7.azuredatabricks.net/browse/folders/4213691640199507?o=2220072380334347) for more detailed reference guides into how the service has been set up at DfE, including more information on workspaces, environments, notebooks, clusters, catalogs etc. Please note that you will need a Databricks account to access these. 
+
+From the [ADA homepage](https://analytical-data-access.education.gov.uk/) you will be able to find data and access cloud analytical tools. This includes:
+
+* Navigation to the Data Discovery Platform to find information regarding metadata of the datasets, including how to request access and information about the quality of the data. The catalogue will eventually include all Department data, as well as tagging of datasets available through ADA.
 
-As long as you're connected to the network - it's available to anyone who works for the Department of Education.
+* Extension of the current platform in the coming months to include POSIT workbench, which will allow R Studio to be used in the cloud.
```

Comment:
Is this definite?

Reply:
also this misses the general analysis workbench bit which is a big part of the ADA thing, should it be something like 

* An analysis workbench, with access to cloud computing / databricks etc. This will be extended to include posit workbench...

Reply:
Think this was my suggestion, though I think the text it was replacing before was different to what it looks like it's replacing now in Git. Agree with Laura that this sits a bit odd here and would probably be better introduced as an analysis workbench. 

I would be vague on the timeline / not mention 'coming months' if we're not certain when it'll be

Reply:
Changed back to analysis workbench, added link to ADA analysis workbench page and removed reference to any timeframes